### PR TITLE
[v6r15] Added consistency checks for DIRAC

### PR DESCRIPTION
**Pull request by marcelovilaca**

### CODE REVIEW

File: DataManagementSystem/Client/ConsistencyChecks.py [hunk 1]
```diff
@@ -0,0 +1,843 @@
...
+    """ c'tor
+        interactive: Data Manager (True) or DIRAC Agente (False)
+        transClient: TransformationClient() if None, else transClient params
+        dm: DataManager() if None, else dm params
+        fc: FileCatalog() if None, else fc params
+        One object for every production/directoriesList...
+    """
+    self.interactive = interactive
+    self.transClient = TransformationClient() if transClient is None else transClient
+    self.dm = DataManager() if dm is None else dm
```
Comment (chaen) [inline DataManagementSystem/Client/ConsistencyChecks.py]: self.dm = dm if dm else DataManager()

**Comment by atsareg** [comment]
I would still suggest to rename the class from ConsistencyChecks to DataConsistencyChecker ( or Validator or Inspector or ... )

**Comment by atsareg** [comment]
You should not add unrelated commits to a PR devoted to another feature. PRs are not just bags where you can put whatever assorted stuff. Be careful next time.

What about renaming the ConsistencyChecks class ?

**Comment by atsareg** [comment]
Since you did not do it yourself, I have renamed the ConsistencyChecks class to ConsistencyInspector

**Comment by marcelovilaca** [comment]
OK, I was working on it today, renaming to DataConsistencyChecker, but there are a lot of imports to fix, sorry for the delay.

**Pull request merged**